# Local Position Setpoints [x, y, z]
waypoints:
  number: 4
  point_1: [-50, 90, 50]
  point_2: [90, 90, 50]
  point_3: [90, 40, 50]
  point_4: [-50, 40, 50]

# clockwise: positive, range: 0-359, units: [degree]
north_pole_diff_angle: 218
# for plane > 20 m
distance_threshold: 24
# system of fcu: apm or px4
fcu_sys: "px4"